examples/pymc3_howto/LKJ_correlation.py

```diff
@@ -53,7 +53,7 @@ def run(n=1000):
         n = 50
     with model:
         trace = pm.sample(n)
-    pm.traceplot(
+    az.plot_trace(
         trace, varnames=["mu", "r"], lines={"mu": mu_r, "r": corr_r[np.triu_indices(n_var, k=1)]}
```

Comment:
I am quite sure arviz does not accept `varnames`, it should be `var_names`

Reply:
Ah, correct! Thanks for that. I updated to arviz `0.11`, and got this error

```
y:212: UserWarning: A valid var_name should be provided, found {'m'} expected from {'mu', 'r'}
  warnings.warn(
Traceback (most recent call last):
  File "LKJ_correlation.py", line 62, in <module>
    run()
  File "LKJ_correlation.py", line 56, in run
    az.plot_trace(
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/traceplot.py", line 238, in plot_trace
    axes = plot(**trace_plot_args)
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/backends/matplotlib/traceplot.py", line 384, in plot_trace
    for _, _, vlines in (j for j in lines if j[0] == var_name and j[1] == selection):
  File "/Users/CloudChaoszero/opt/anaconda3/envs/pymc3-dev-py38/lib/python3.8/site-packages/arviz/plots/backends/matplotlib/traceplot.py", line 384, in <genexpr>
    for _, _, vlines in (j for j in lines if j[0] == var_name and j[1] == selection):
IndexError: string index out of range
```
(This error looks related https://github.com/arviz-devs/arviz/issues/988) 

Any thoughts why this happened?

Reply:
I think I found out why now. It's due to the `lines` param.

I removed it to make the `az.plot_trac()` work.

Reply:
Try

    lines=(("mu", {}, mu_r), ("r", {}, corr_r[np.triu_indices(n_var, k=1)]))